src/Uno.Extensions.Reactive/Core/Axes/IChangeSet.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using Uno.Extensions.Collections;
+
+namespace Uno.Extensions.Reactive
+{
+	internal interface IChangeSet : IEnumerable<IChange>
+	{
+	}
+
+	//internal interface IChangeSet<out TChange> : IChangeSet, IEnumerable<TChange>
```

Comment:
Remove ?

Reply:
This has been removed in my working branch